uber/templates/regform.html

```diff
@@ -91,6 +91,14 @@
         });
     {% endif %}
 
+    $(function () {
+        staffingClicked();
+        if ($.field('staffing')) {
+            {% if attendee.shifts %} $.field('staffing').prop('readonly', true).prop('title', 'Please {{ "contact your department head to drop shifts" if c.AT_THE_CON else "unassign yourself from shifts" }} before changing your volunteer status.'); {% endif %}
```

Comment:
Yeah, I expect STOPS would prefer to be able to simply uncheck the checkbox, since that's what we've been able to do in past years. We can ask them, though.